### ISSUE TITLE
Shader Minifier Doesn't Minify Functions Correctly

### ISSUE THREAD

**Issue by tomlister** [ComunidadAylas/PackSquash#43]
**Describe the bug**
When minifying a shader that contains a function with arguments, the argument type and name are merged, causing an error when loaded.
```glsl
vec3 hash(vec3 ar){return ar;}
```

**To Reproduce**
Steps to reproduce the behaviour:

Create a function with arguments and minify.
i.e
```glsl
vec3 hash (vec3 ar) {
   return ar;
}
```

**Expected behaviour**
The argument type and name should be separated by whitespace.
```glsl
vec3 hash(vec3 ar){return ar;}
```

**Attached files**
N/A

**Additional context**
N/A

**Comment by AlexTMjugador** [comment]
Thank you for your bug report! I hope this didn't cause much trouble, as it can be a bit painful to debug. However, this particular issue has been reported and fixed shortly after the v0.3.0 release, without an issue being created for it. The commit that fixed this issue is this one: https://github.com/ComunidadAylas/PackSquash/commit/d2dda845351176d01afa3a93e158404ebd555279. The latest unstable builds can be downloaded from the [actions page](https://github.com/ComunidadAylas/PackSquash/actions), or, if you're using the GitHub action, you can also change the `packsquash_version` parameter to `latest`.

v0.3.1 will contain this and other fixes and changes, according to the [roadmap](https://github.com/ComunidadAylas/PackSquash/projects/1).

**Comment by tomlister** [comment]
Cheers 🥂
